ddl/schematracker/checker.go

```diff
@@ -62,13 +63,32 @@ func (d *Checker) Enable() {
 	d.closed = false
 }
 
+// CreateTestDB creates a `test` database like the default behaviour of TiDB.
+func (d Checker) CreateTestDB() {
```

Comment:
Could move `CreateTestDB` and `createTestDB` into `NewSchemaTracker` function?

Reply:
in normal DM using case, `test` database is not wanted.